drawBot/drawBotDrawingTools.py

```diff
@@ -405,10 +409,6 @@ def saveImage(self, paths, multipage=None, **options):
             context = getContextForFileExt(ext)
             if context is None:
                 raise DrawBotError("Could not find a supported context for: '%s'" % ext)
-            allowedSaveImageOptions = set(optionName for optionName, optionDoc in context.saveImageOptions)
-            for optionName in options:
-                if optionName not in allowedSaveImageOptions:
-                    warnings.warn("Unrecognized saveImage() option found for %s: %s" % (context.__class__.__name__, optionName))
```

Comment:
A) I don't care much for the path = list option. I never use it. I find it much easier to use separate saveImage() calls
B) I did it inside the loop so that the warning message is specific for the context being used. To specify `ffmpegCodec` when saving `.png` should cause a warning, but not when saving `.mp4`. Please revert.

Reply:
A) I understand :)

B) fe: when setting a different `imagePNGGamma` for generated png's to build a  mp4 would show me a warning?

I guess either solution are optimal as saveImage is maybe combining to much :)

Reply:
Good point, but perhaps the mp4 context should then list those options as well.